### PR TITLE
Updated the Changelog and README for newer version

### PR DESCRIPTION
**Pull request by munavvar20**
@ask I have updated the Changelog file as well as README for the newer release.

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -4,6 +4,19 @@
  Change history
 ================
 
+.. _version-1.0.2:
+
+1.0.2
+=====
+:release-date: 2018-07-25 08:30 pm. CST
+:release-by: Munavvar Khan
```
Comment (auvipy) [inline Changelog]: you are not going to release this so remove your name and date.

**Comment by jedie** [comment]
1.0.2 is not on PyPi :(

**Comment by auvipy** [comment]
or this should be 1.1.0?

**Comment by DZPM** [comment]
There is no github tag/release, either: https://github.com/celery/django-celery-results/releases

Looking forward to a new pypi release. Thanks!

**Comment by munavvar20** [comment]
Any updates on this release?

**Pull request merged**